soc/xtensa/intel_adsp/common/clk.c

```diff
@@ -13,53 +13,53 @@
 #include <adsp-clk.h>
 #include <adsp_shim.h>
 
-static struct cavs_clock_info platform_clocks[CONFIG_MP_MAX_NUM_CPUS];
+static struct adsp_clock_info platform_clocks[CONFIG_MP_MAX_NUM_CPUS];
 static struct k_spinlock lock;
 
-int cavs_clock_freq_enc[] = CAVS_CLOCK_FREQ_ENC;
+int adsp_clock_freq_enc[] = ADSP_CLOCK_FREQ_ENC;
 #ifndef CONFIG_SOC_INTEL_CAVS_V15
-int cavs_clock_freq_mask[] = CAVS_CLOCK_FREQ_MASK;
+int adsp_clock_freq_mask[] = ADSP_CLOCK_FREQ_MASK;
 #endif
 
 #ifdef CONFIG_SOC_INTEL_CAVS_V15
 static void select_cpu_clock_hw(uint32_t freq)
 {
 	uint8_t cpu_id = _current_cpu->id;
-	uint32_t enc = cavs_clock_freq_enc[freq] << (8 + cpu_id * 2);
+	uint32_t enc = adsp_clock_freq_enc[freq] << (8 + cpu_id * 2);
 	uint32_t mask = CAVS_CLKCTL_DPCS_MASK(cpu_id);
 
-	CAVS_SHIM_CLKCTL &= ~CAVS_CLKCTL_HDCS;
-	CAVS_SHIM_CLKCTL = (CAVS_SHIM_CLKCTL & ~mask) | (enc & mask);
+	ADSP_CLKCTL &= ~CAVS_CLKCTL_HDCS;
```

Comment:
@abonislawski (https://github.com/zephyrproject-rtos/zephyr/pull/51919->CAVS_CLKCTL_HDCS?)
CAVS_CLKCTL_* definitions have not been renamed... Should I update/rename/cleanup it too or they can be skied at this moment?